src/components/Selectors/RadioButton/RadioButton.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { FC, useEffect } from 'react';
+import { RadioButtonProps } from '../';
+import { mergeClasses } from '../../../shared/utilities';
+import { useRadioGroup } from './RadioGroup.context';
+
+import styles from './radio.module.scss';
+
+export const RadioButton: FC<RadioButtonProps> = ({
+    ariaLabel,
+    activeRadioButton,
+    checked = false,
+    disabled = false,
+    name,
+    value = '',
+    id = `radiobox-${value}-${Math.random().toString(36).slice(2)}`,
+}) => {
+    const { onRadioButtonClick, currentRadioButton } = useRadioGroup();
+    const isActive: boolean = value === currentRadioButton;
+
+    useEffect(() => {
+        if (onRadioButtonClick) onRadioButtonClick(activeRadioButton, null);
```

Comment:
Need to somehow update `focus()` on the `activeRadioButton`, but from what I've read it's a index, so will also need a pointer to the `<span className={radioButtonClassNames} tabIndex={0}></span>` and update focus as the arrow keys are used.